core/src/test/scala/org/apache/spark/SparkConfSuite.scala

```diff
@@ -354,6 +354,16 @@ class SparkConfSuite extends SparkFunSuite with LocalSparkContext with ResetSyst
     }
   }
 
+  test("SPARK-27244 toDebugString should redact passwords") {
+    val EXPECTED_LOG = s"dummy.password=${Utils.REDACTION_REPLACEMENT_TEXT}"
```

Comment:
No capital letters just camelCase and move as close to usage as possible. Actually I would inline this one too.

Reply:
ok doing this change too, thank @attilapiros